tzlocal/darwin.py

```diff
@@ -27,7 +26,7 @@ def __exit__(self, type, value, traceback):
     from subprocess import Popen
 
 
-def _get_localzone(_root='/'):
+def get_localzone_name(_root='/'):
```

Comment:
No, this function must return a timezone, as the configured timezone doesn't have to be a name.

Reply:
The purpose of the refactor was to move some generalized functionality from the system specific modules to a general scope, which currently is `tzlocal.__init__`

Reply:
If you look below, the only change is that it now doesn't call `pytz.timezone`, it seemed more uniform to the overall way that the system-specific modules behaved

Reply:
Yes, but these functions must return a timezone, as the configure time zone might not have a name.